model/pdata/generated_common_alias.go

```diff
@@ -19,18 +19,31 @@ package pdata
 
 import "go.opentelemetry.io/collector/pdata/pcommon"
 
-// InstrumentationScope is an alias for pcommon.InstrumentationScope struct.
+// InstrumentationScope is a message representing the instrumentation scope information.
+//
+// This is a reference type, if passed by value and callee modifies it the
+// caller will see the modification.
+//
+// Must use NewInstrumentationScope function to create new instances.
+// Important: zero-initialized instance is not valid for use.
```

Comment:
Don't need to update the deprecated once.

Reply:
This is generated code. It would require some effort to split the generator to produce different godoc strings between the old and new aliases.

Reply:
I think you can make the "generator" no longer generate the `model/pdata` directory. It is gone :)

Reply:
You're right. This is easy to do :)

Reply:
done